node_modules/oae-core/filepreview/filepreview.html

```diff
@@ -54,6 +54,32 @@
     </div>
 --></div>
 
+<!-- VIDEO PREVIEW -->
+<div id="filepreview-video-template"><!--
+    <div id="filepreview-video-container" class="text-center">
+        {if content.previews && content.previews.video}
+            {var url = content.previews.video.replace('remote:', '')}
+            {var mime = 'video/mp4'}
+        {else}
+            {var url = content.downloadPath}
```

Comment:
The idea was that if for some reason Zencoder encoding failed, if the video happened to be in one of the HTML5/Flash supported formats we could still play it. I could flesh this out a bit to check the mime-type and whether it is one of them, but I thought it would be a good fallback even if other formats are not as widely supported as mp4.